packages/engine/Specs/Scene/VoxelBoxShapeSpec.js

```diff
@@ -829,43 +829,4 @@ describe("Scene/VoxelBoxShape", function () {
       );
     }).toThrowDeveloperError();
   });
-
```

Comment:
Is there a relevant replacement spec for the Jacobian approach? (It's OK if there's not.)

Reply:
No, all step size math has been moved to the shader.